content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -204,6 +204,104 @@ ALTER TABLE "Profile" ADD FOREIGN KEY("userId")REFERENCES "User"("id") ON DELETE
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'mssql']}>
+
+<TabbedContent tabs={[<FileWithIcon text="SQL" icon="code"/>, <FileWithIcon text="Tables" icon="database"/>]}>
+
+<tab>
+
+```sql
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- CreateTable
+CREATE TABLE [dbo].[Post] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [createdAt] DATETIME2 NOT NULL CONSTRAINT [DF__Post__createdAt] DEFAULT CURRENT_TIMESTAMP,
+    [updatedAt] DATETIME2 NOT NULL,
+    [title] VARCHAR(255) NOT NULL,
+    [content] NVARCHAR(1000),
+    [published] BIT NOT NULL CONSTRAINT [DF__Post__published] DEFAULT 0,
+    [authorId] INT NOT NULL,
+    CONSTRAINT [PK__Post__id] PRIMARY KEY ([id])
+);
+
+-- CreateTable
+CREATE TABLE [dbo].[Profile] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [bio] NVARCHAR(1000),
+    [userId] INT NOT NULL,
+    CONSTRAINT [PK__Profile__id] PRIMARY KEY ([id]),
+    CONSTRAINT [Profile_userId_unique] UNIQUE ([userId])
+);
+
+-- CreateTable
+CREATE TABLE [dbo].[User] (
+    [id] INT NOT NULL IDENTITY(1,1),
+    [email] NVARCHAR(1000) NOT NULL,
+    [name] NVARCHAR(1000),
+    CONSTRAINT [PK__User__id] PRIMARY KEY ([id]),
+    CONSTRAINT [User_email_unique] UNIQUE ([email])
+);
+
+-- AddForeignKey
+ALTER TABLE [dbo].[Post] ADD CONSTRAINT [FK__Post__authorId] FOREIGN KEY ([authorId]) REFERENCES [dbo].[User]([id]) ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE [dbo].[Profile] ADD CONSTRAINT [FK__Profile__userId] FOREIGN KEY ([userId]) REFERENCES [dbo].[User]([id]) ON DELETE CASCADE ON UPDATE CASCADE;
+
+COMMIT TRAN;
+
+END TRY
+BEGIN CATCH
+
+IF @@TRANCOUNT > 0
+BEGIN 
+    ROLLBACK TRAN;
+END;
+THROW
+
+END CATCH
+```
+
+</tab>
+
+<tab>
+
+**User**
+
+| Column name | Type      | Primary key | Foreign key | Required | Default            |
+| :---------- | :-------- | :---------- | :---------- | :------- | :----------------- |
+| `id`        | `INTEGER` | **✔️**      | No          | **✔️**   | _autoincrementing_ |
+| `name`      | `TEXT`    | No          | No          | No       | -                  |
+| `email`     | `TEXT`    | No          | No          | **✔️**   | -                  |
+
+**Post**
+
+| Column name | Type           | Primary key | Foreign key | Required | Default            |
```

Comment:
This table is missing `updatedAt` (might also apply to other databases if this was copied from somewhere else)